controllers/glanceapi_controller.go

```diff
@@ -758,7 +759,7 @@ func (r *GlanceAPIReconciler) reconcileNormal(
 		}
 	}
 	// iterate over availableBackends for backend specific cases
-	for i := 0; i < len(availableBackends); i++ {
+	for i := range len(availableBackends) {
```

Comment:
I am not sure but range here will not work as len(availableBackends) will return integer.
This should be like

```
for i := range availableBackends {
    backendToken := strings.SplitN(availableBackends[i], ":", 2)
    switch backendToken[1] {
    case "cinder":
        // your code here
    }
}
```